src/docs/loginRouter.docs.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @openapi
+ * tags:
+ *  name: Authentication
+ *  description: Routes for authentication
+ */
+
+/**
+ * @swagger
+ * /api/v1/logout:
+ *  get:
+ *      summary: Logout from phantom
+ *      description: The
+ *      tags:
+ *          - Logout
+ *      responses:
+ *          200:
+ *              description: successfully logged out
+ */
+
+/**
+ * @swagger
+ * /api/v1/login:
+ *  post:
+ *      summary: Login into phantom
+ *      description: The user is able to login into phantom
+ *      tags:
+ *          - Login
+ *      requestBody:
+ *          required: true
+ *          content:
+ *              application/json:
+ *                  schema:
+ *                      properties:
+ *                          email:
+ *                              required: true
+ *                              type: string
+ *                              example: test@me.com
+ *                          password:
+ *                              required: true
+ *                              type: string
+ *                              example: 1234
+ *
+ *      responses:
+ *          200:
+ *              description: login is successful
+ *          404:
+ *              description: user does not exist
+ *          401:
```

Comment:
This response should have a code of **400** since the user is not being denied access but is sending wrong data to the server.

401 and 403 are reserved for Authorization related errors.